modules/mobile/config/routes.rb

```diff
@@ -6,4 +6,8 @@
   namespace :v0 do
     get '/user', to: 'users#show'
   end
+
+  namespace :v0 do
+    get '/military-service-history', to: 'military_information#get_service_history'
```

Comment:
Can this route be moved underneath line 7?

Reply:
Done